### PR TITLE
2016 Prius DBC

### PR DESCRIPTION
**Pull request by vanillagorillaa**
Submitting DBC for @andy867

### CODE REVIEW

File: toyota_prius_2016_pt.dbc [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+ SG_ UNITS : 26|2@0+ (1,0) [0|3] "" XXX
+
+BO_ 1568 SEATS_DOORS: 8 XXX
+ SG_ SEATBELT_DRIVER_UNLATCHED : 62|1@0+ (1,0) [0|1] "" XXX
+ SG_ DOOR_OPEN_FL : 45|1@0+ (1,0) [0|1] "" XXX
+ SG_ DOOR_OPEN_RL : 42|1@0+ (1,0) [0|1] "" XXX
+ SG_ DOOR_OPEN_RR : 43|1@0+ (1,0) [0|1] "" XXX
+ SG_ DOOR_OPEN_FR : 44|1@0+ (1,0) [0|1] "" XXX
+
+BO_ 528 TRACK_A_0: 8 XXX
```
Comment (energee) [inline toyota_prius_2016_pt.dbc]: Tracks on Powertrain can?

Comment (energee) [reply toyota_prius_2016_pt.dbc]: Looks like the 2017 dbc should work fine for this car

Comment (vanillagorillaa) [reply toyota_prius_2016_pt.dbc]: I believe @andy867 Is wanting the PR to resolve Chffr Metrics data which is different on the 16 than the 17 model.

Comment (rbiasini) [reply toyota_prius_2016_pt.dbc]: mmm... chffr metric line seems the same as 2017 file, isn't it? Also, as mentioned by @energee , I would be surprised if powertrain data and radar tracks were on the same bus.

**Comment by vanillagorillaa** [comment]
Closing and will open a new PR with changes

**Pull request closed without merging**